Formula/opencv.rb

```diff
@@ -89,6 +91,20 @@ def install
       -DPYTHON3_EXECUTABLE=#{Formula["python@3.9"].opt_bin}/python3
     ]
 
+    # Disable precompiled headers and force opencv to use brewed libraries on Linux
+    on_linux do
+      args << "-DENABLE_PRECOMPILED_HEADERS=OFF"
+      args << "-DJPEG_LIBRARY=#{Formula["libjpeg"].opt_lib}/libjpeg.so"
+      args << "-DOpenBLAS_LIB=#{Formula["openblas"].opt_lib}/libopenblas.so"
+      args << "-DOPENEXR_ILMIMF_LIBRARY=#{Formula["openexr"].opt_lib}/libIlmImf.so"
```

Comment:
Just noting that we may need to recheck this CMake variable/value.
`libIlmImf` was changed to `libOpenEXR` in v3 release of OpenEXR.

I did see the Linux bottle link to `libOpenEXR-3_1.so.30`, so it may be able to autodetect it.